### DISCUSSION TITLE
How to use darkreader without CDN?

### DISCUSSION THREAD

**Discussion by MikeK184** [darkreader/darkreader#10144] [General]
Hi there,

I've recently added darkreader to my wordpress page through CDN successfully with adding the following code to `functions.php` in my theme:

```php
function darkreader() {
    ?>
        <script src="https://cdn.jsdelivr.net/npm/darkreader/darkreader.js"></script>
        <script type="text/javascript">
            DarkReader.setFetchMethod(window.fetch)
            DarkReader.enable({
                //brightness: 100,
                //contrast: 90,
                //sepia: 10
                });
        </script>
    <?php
}
add_action('wp_head', 'darkreader');
```

However I would like to download the file and serve it locally but unfortunately I just couldn't get it to work. However I've set up prismjs sucessfully locally:
```php
function add_prism() {
    wp_enqueue_style( 'prism-css',  get_stylesheet_directory_uri() . '/prism.css' );
    wp_enqueue_script( 'prism-js',  get_stylesheet_directory_uri() . '/prism.js', [], false, true );
}
add_action( 'wp_enqueue_scripts', 'add_prism' );
```

But replicating the that didn't work out for me and crashed my page, so I had to revert my changes to the CDN method.

**Comment by Gusted** [comment]
> But replicating the that didn't work out for me and crashed my page, so I had to revert my changes to the CDN method.

Could you elaborate a bit more on this? Currently there's not much actionable information to go on, to help you.
